app/serializers/comment.js

```diff
@@ -7,6 +7,8 @@ export default ApplicationSerializer.extend({
     repliesCount: { serialize: false },
     createdAt: { serialize: false },
     editedAt: { serialize: false },
-    updatedAt: { serialize: false }
+    updatedAt: { serialize: false },
+
+    uploads: { serialize: true }
```

Comment:
Is this needed at all? Should serialize all except ones explicitly defined not to.